### PR TITLE
Add filters

### PR DESCRIPTION
**Pull request by igormukhingmailcom**

### CODE REVIEW

File: src/Doctrine/ORM/GiftCardRepository.php [hunk 1]
```diff
@@ -12,11 +12,14 @@
 use Sylius\Component\Core\Model\CustomerInterface;
 use Sylius\Component\Core\Model\OrderItemUnitInterface;
 
-final class GiftCardRepository extends EntityRepository implements GiftCardRepositoryInterface
+/** not final */ class GiftCardRepository extends EntityRepository implements GiftCardRepositoryInterface
```
Comment (loevgaard) [inline src/Doctrine/ORM/GiftCardRepository.php:15]: This was an error in the first place. Just remove this comment. We won't let it happen again :D

File: src/Grid/FieldTypes/StringFieldType.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        try {
+            $value = $this->propertyAccessor->getValue($data, $field->getPath());
+        } catch (UnexpectedTypeException $e) {
+            return '';
+        }
+
+        return htmlspecialchars((string) $value);
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
```
Comment (loevgaard) [inline src/Grid/FieldTypes/StringFieldType.php:34]: Is this empty method needed / is it in the interface?

File: src/Resources/config/services/field_types.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults public="true" />
```
Comment (loevgaard) [inline src/Resources/config/services/field_types.xml:5]: Should this be public?

**Review by loevgaard — CHANGES_REQUESTED** [review]

**Review by loevgaard — APPROVED** [review]

File: src/Grid/FieldTypes/StringFieldType.php (hunk 2 above)
Comment (igormukhingmailcom) [reply src/Grid/FieldTypes/StringFieldType.php:34]: Yes, it in interface

File: src/Resources/config/services/field_types.xml (hunk 3 above)
Comment (igormukhingmailcom) [reply src/Resources/config/services/field_types.xml:5]: In original `vendor/sylius/grid-bundle/src/Bundle/Resources/config/services/field_types.xml` we have `<defaults public="true" />` too, but I've tried and looks like it works without that

**Pull request merged**